### DISCUSSION TITLE
Next release coming soon -- please translate

### DISCUSSION THREAD

**Discussion by kebekus** [Akaflieg-Freiburg/enrouteText#46] [Team Posts]
Dear Translators,

a new release is coming up early next week, which will bring substantial simplifications in the "Map & Data" page. Could you please update the translations? I see that one (=1) string is still missing in French, Italian and Polish.

Best wishes · Thank you for your help,

Stefan.

**Comment by Ivan081** [comment]
Where I don't find untranslated

Il ven 21 ott 2022, 07:28 Stefan Kebekus ***@***.***> ha
scritto:

**Comment by TTC70** [comment]
Ivan,
I did it this morning 👍👍

Il Ven 21 Ott 2022, 09:27 Ivan ***@***.***> ha scritto:

**Comment by kebekus** [comment]
Thanks!
